qemu/qemu.go

```diff
@@ -1162,9 +1175,13 @@ type Knobs struct {
 	// However the setup is different from normal pre-allocation.
 	// Hence HugePages has precedence over MemPrealloc
 	// HugePages will pre-allocate all the RAM from huge pages
+	//
+	// Deprecated (use Memory.HugePages instead)
 	HugePages bool
```

Comment:
Maybe you could have removed that if that's deprecated. It would be less confusing to use.